README.md

```diff
@@ -56,7 +53,7 @@ sudo apt-get install python3-numpy python3-pytest
 Using pip you can install the required dependencies for pymavlink :
 
 ```bash
-sudo python -m pip install --upgrade future lxml
```

Comment:
I believe these were deliberately `sudo` to install them for the entire system rather than just the current user. 

*I* install just for the local user - but not everybody does that.

I don't see any reason to conflate other Py2-related changes in this PR with changing these instructions..

Reply:
Here's the warning pip gives when I install with sudo:

> WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv

It's a bad recommendation here to use `sudo pip install`. It should be either non-sudo or installed with the package manager as `apt install python3-lxml python3-future`. The reason this is important in this PR is if the user has multiple python environments (e.g. installed with `pyenv`), `sudo` is going to change the variable environment, likely causing the package to be installed in the wrong place.